src/components/OurWork.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function OurWork() {
+  return (
+    <div>
+      <h1>We're Sorry!</h1>
+      <p>
+        <span>Note:</span>
+        This page is down for maintenance. We are working to get it back up and
+        running as soon as possible. Please check back!
+      </p>
+    </div>
+  );
+}
```

Comment:
Here, when I click on a button in the navbar, I would expect not the see the search bar. What do you think about a page which does not include the search bar the the main title & subtitle ?

Reply:
I'm not sure! What if we keep the main title and subtitle, and remove just the search bar. I also don't have a design in mind! Any suggestions ??